modules/setting/setting.go

```diff
@@ -1606,7 +1606,7 @@ func newWebhookService() {
 	Webhook.QueueLength = sec.Key("QUEUE_LENGTH").MustInt(1000)
 	Webhook.DeliverTimeout = sec.Key("DELIVER_TIMEOUT").MustInt(5)
 	Webhook.SkipTLSVerify = sec.Key("SKIP_TLS_VERIFY").MustBool()
-	Webhook.Types = []string{"gitea", "gogs", "slack", "discord", "dingtalk"}
+	Webhook.Types = []string{"gitea", "gogs", "slack", "discord", "dingtalk","typetalk"}
```

Comment:
```suggestion
	Webhook.Types = []string{"gitea", "gogs", "slack", "discord", "dingtalk", "typetalk"}
```

Reply:
@adelowo Thanks! I committed the change.